src/utility/espspi_proxy.h

```diff
@@ -371,7 +371,7 @@ class EspSpiProxy
      */
     uint8_t crc8(uint8_t *buffer, uint8_t bufLen)
     {
-        const static uint8_t POLY = 0x07;
+//        const static uint8_t POLY = 0x07;
```

Comment:
Thanks, it is a Copy-Paste issue. But next time, please, open another PR for changes not related to the original PR.

Reply:
Sorry, don't know why it didn't end up in a separate pull request, sorry..

Reply:
Please, delete the commented lines completely, there is no reason for them to remain here. Thanks.

Reply:
Done.

Reply:
I don't see it, did you commit the change?

Reply:
Oops, didn't see the "delete the commented lines" comment, so this is not reflected yet, will do so now.

Reply:
Done, sorry for the typo, should read "unused variables" not "unused libraries".